src/audio/multiband_drc/multiband_drc.c

```diff
@@ -338,6 +339,32 @@ static int multiband_drc_cmd_get_data(struct comp_dev *dev,
 	return ret;
 }
 
+static int multiband_drc_cmd_get_value(struct comp_dev *dev,
+				       struct sof_ipc_ctrl_data *cdata)
+{
+	struct multiband_drc_comp_data *cd = comp_get_drvdata(dev);
+	int j;
+	int ret = 0;
+
+	switch (cdata->cmd) {
+	case SOF_CTRL_CMD_SWITCH:
+		comp_dbg(dev, "multiband_drc_cmd_get_value(), SOF_CTRL_CMD_SWITCH");
+		for (j = 0; j < cdata->num_elems; j++) {
+			cdata->chanv[j].channel = j;
+			cdata->chanv[j].value = cd->process_enabled;
```

Comment:
do we want to make this per channel?

Reply:
I think we only want this as one flag, not per channel. Since I'm not familiar to this I just copied the code from volume.c for now. I will update this after I could build and run the switch.